lib/airbrake.rb

```diff
@@ -19,3 +19,28 @@
 require 'airbrake/resque/failure' if defined?(Resque)
 require 'airbrake/sidekiq/error_handler' if defined?(Sidekiq)
 require 'airbrake/delayed_job/plugin' if defined?(Delayed)
+
+##
+# This module defines the Airbrake API and serves as a namespace for other
+# classes and modules.
+##
+module Airbrake
+  class << self
+    ##
+    # Allows users to add their own notice's builders. For example, if it's needed to
+    # add a REMOTE_IP to the notice's params it can be easily done in standard
+    # initializer for Airbrake.
+    #
+    # @example Ignore all notices
+    #   Airbrake.add_rack_builder |notice, request|
+    #     notice[:params][:remoteIp] = request.env['REMOTE_IP']
+    #   end
+    #
+    # @yield [notice, request] The builder block
+    # @note Once a builder was added, there's no way to delete it
```

Comment:
I would remove this because you surely can delete them with `Airbrake::Rack::NoticeBuilder.builders.clear` :)